extensions/amp-list/0.1/amp-list.js

```diff
@@ -713,9 +749,9 @@ export class AmpList extends AMP.BaseElement {
       .then(elements => this.render_(elements, current.append));
     if (!isSSR) {
       const payload = /** @type {!JsonObject} */ (current.payload);
-      renderPromise = renderPromise
-        .then(() => this.maybeRenderLoadMoreTemplates_(payload))
-        .then(() => this.maybeSetLoadMore_());
+      renderPromise = renderPromise.then(() =>
+        this.maybeRenderLoadMoreTemplates_(payload)
```

Comment:
Does it matter that the order changed here?

Reply:
This change is intentional. Basically we only ever want to call `maybeSetLoadMore_` once. This code is responsible for setting up auto-load-more or showing the load-more button the first time round if a load-more-src is present. The previous implementation was incorrect and resulted in this function being needlessly called on every fetch (it was idempotent so that was fine, but might as well just not have duplicate calls to it).